apps/api/src/app/events/dtos/trigger-event-request.dto.ts

```diff
@@ -77,6 +69,7 @@ export class TriggerEventRequestDto {
         $ref: getSchemaPath(TopicPayloadDto),
       },
     ],
+    // type: [String, SubscriberPayloadDto, TopicPayloadDto],
```

Comment:
Is this meant to be commented out or not?

Reply:
Yes to test CI for now, will put it back soon.